nuget/OpenSans/Uno.Fonts.OpenSans/Uno.Fonts.OpenSans.csproj

```diff
@@ -0,0 +1,65 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
+	<PropertyGroup>
+		<TargetFrameworks>uap10.0.18362;netstandard2.0;net6.0-windows10.0.18362</TargetFrameworks>
```

Comment:
This was a copy form Roboto packaging. However, fluent packaging only produces netstandard1.0 in the package.

@jeromelaban @kazo0 Why are Roboto and Fluent different in packaging? Which way should I use for OpenSans? I'm leaning towards the way we package Fluent.

Reply:
I adjusted the new package to only have netstandard1.0